package.json

```diff
@@ -40,15 +40,17 @@
   "scripts": {
     "build:site": "cd site && npm ci && npm run build",
     "start": "node ./bin/run",
-    "test": "run-s test:*",
-    "ava": "ava --verbose",
-    "format": "prettier --write \"{src,scripts,site,site/src,tests,.circleci,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\"",
-    "test:lint": "eslint --fix \"{.github,src,scripts,site,tests}/**/*.{js,md,html}\"",
-    "test:format": "prettier --check \"{src,scripts,site,site/src,tests,.circleci,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\"",
-    "test:cli-version": "npm run start -- --version",
-    "test:cli-help": "npm run start -- --help",
-    "test:install-deps": "npm ci --prefix tests/site-cra",
-    "test:ava": "nyc --reporter=lcov ava --verbose && nyc report",
+    "test": "npm run format && npm run test:dev",
+    "format": "run-s format:*",
+    "format:lint": "eslint --fix \"{.github,src,scripts,site,tests}/**/*.{js,md,html}\"",
```

Comment:
`test:lint` is renamed to `format:lint` since both ESLint and Prettier are skipped in Node 8 in CI, and they are both somewhat related.